modules/osd-monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * monitoring/osd-managing-metrics.adoc
+
+[id="querying-metrics-for-user-defined-projects-as-a-developer_{context}"]
+= Querying metrics for user-defined projects as a developer
+
+You can access metrics for a user-defined project as a developer or as a user with view permissions for the project.
+
+In the *Developer* perspective, the Metrics UI includes some predefined CPU, memory, bandwidth, and network packet queries for the selected project. You can also run custom Prometheus Query Language (PromQL) queries for CPU, memory, bandwidth, network packet and application metrics for the project.
+
+[NOTE]
+====
+Developers can only use the *Developer* perspective and not the *Administrator* perspective. As a developer you can only query metrics for one project at a time. Developers cannot access the third-party UIs provided with {product-title} monitoring.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a developer or as a user with view permissions for the project that you are viewing metrics for.
+* You have enabled monitoring for user-defined projects.
+* You have deployed a service in a user-defined project.
+* You have created a `ServiceMonitor` custom resource definition (CRD) for the service to define how the service is monitored.
+
+.Procedure
+
+. From the *Developer* perspective in the {product-title} web console, select *Monitoring* -> *Metrics*.
```

Comment:
select *Monitoring*, there is no -> *Metrics*.

Reply:
Changed it.

Reply:
Actually, I changed it back to keep Metric because Junqi checked and said:

select Monitoring > Metrics
"Metrics" is a tab here, see image in the conversation
https://github.com/openshift/openshift-docs/pull/30483/#discussion_r597619145